### PR TITLE
docs: Fix invalid URLs

### LINKED ISSUE #149: Fix broken documentation URLs
Correct the document URLs which have gone stale.

Related: https://github.com/kata-containers/tests/pull/398

### PR DESCRIPTION
**Pull request by jodh-intel**
Correct invalid document URLs.

Fixes #149.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Related:

- https://github.com/kata-containers/runtime/pull/377
- https://github.com/kata-containers/tests/pull/398

Note that #64 is needed to correct the last incorrect URL (see https://github.com/kata-containers/documentation/pull/64#issuecomment-395798813).

File: architecture.md [hunk 1]
```diff
@@ -544,7 +544,7 @@ dedicated master node) calls into a compute kubelet. This kubelet instance is
 responsible for managing the lifecycle of pods within the nodes and eventually relies
 on a container runtime to handle execution. The kubelet architecture decouples
 lifecycle management from container execution through the dedicated
-[`gRPC`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto)
+[`gRPC`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto)
```
Comment (grahamwhaley) [inline architecture.md]: `v1alpha2` - I had to go check, coz that was not inspiring me with confidence :-)

Comment (jodh-intel) [reply architecture.md]: I totally agree, but it looks like that's as good as it gets...

File: zun_kata.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ Containers, start DevStack, and set the environment variables to use
 ```
 sudo mkdir -p /opt/stack
 sudo chown $USER /opt/stack
-git clone https://git.openstack.org/openstack-dev/devstack /opt/stack/devstack
+git clone https://github.com/openstack-dev/devstack /opt/stack/devstack
```
Comment (jodh-intel) [inline zun_kata.md]: @eadamsintel - could you verify these two zun URLs?

**Pull request merged**